docs/en/Overview.md

```diff
@@ -10,76 +10,74 @@ priority: 0
+and is deployed at a wide range of companies.
+It is one of the fastest growing open source projects.
+In a span of three years, Alluxio has attracted more than [800 contributors](https://github.com/alluxio/alluxio/graphs/contributors)
+from over 150 institutions including
+[Alibaba](http://www.alibaba.com),
+[Alluxio](http://www.alluxio.com/),
+[Baidu](https://www.baidu.com),
+[CMU](https://www.cmu.edu/),
+[Google](https://www.google.com),
+[IBM](https://www.ibm.com),
+[Intel](http://www.intel.com/),
+[NJU](http://www.nju.edu.cn/english/),
+[Red Hat](https://www.redhat.com/),
+[UC Berkeley](https://amplab.cs.berkeley.edu/),
+and [Yahoo](https://www.yahoo.com/).
+The project is the storage layer of the Berkeley Data Analytics Stack ([BDAS](https://amplab.cs.berkeley.edu/bdas/))
+and is also part of the [Fedora distribution](https://fedoraproject.org/wiki/SIGs/bigdata/packaging).
+Today, Alluxio is deployed in production by hundreds of organizations and runs on clusters exceeding 1,000 nodes.
 
 <p align="center">
 <img src="{{site.baseurl}}{% link img/stack.png %}" width="800" alt="Ecosystem"/>
 </p>
 
 ## Benefits
 
-Alluxio helps overcome the obstacles to extracting value from data by making it simple to give
-applications access to whatever data is needed, regardless of format or location. The benefits of
-Alluxio Include:
+Alluxio helps overcome the obstacles in extracting insight from data by simplifying how applications
+access their data, regardless of format or location. The benefits of Alluxio include:
 
 * **Memory-Speed I/O**: Alluxio can be used as a distributed shared caching service so that compute
-applications talking to Alluxio can transparently cache frequently accessed data, especially data
+applications talking to Alluxio can transparently cache frequently accessed data, especially
 from remote locations, to provide in-memory I/O throughput.
 
 * **Simplified Cloud and Object Storage Adoption**: Cloud and object storage systems use different
-semantics that have performance implications compared to traditional file systems. For example, when
-accessing data in cloud storage there is no node-level locality or cross-application caching. There
-are also different performance characteristics in common file system operations like directory
-listing (‘ls’) and ‘rename’, which often add significant overhead to analytics. Deploying Alluixo
-with cloud or object storage can close the semantics gap and achieve significant performance gains.
+semantics that have performance implications compared to traditional file systems. Common file
+system operations such as directory listing and renaming often incur significant performance
+overhead. When accessing data in cloud storage, applications have no node-level locality or
+cross-application caching. Deploying Alluxio with cloud or object storage mitigates these problems
+because...TODO: how does Alluxio address this???
```

Comment:
this section explains the differences between object store vs filesystems but does not explain why alluxio helps